doc/source/notebooks/understanding/models.ipynb

```diff
@@ -848,11 +848,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Building new models\n",
+    "## Building new models\n",
     "\n",
-    "To build new models, you'll need to inherit from `gpflow.models.Model`. Parameters are instantiated with `gpflow.Param`. You may also be interested in `gpflow.params.Parameterized` which acts as a 'container' of `Param`s (e.g. kernels are Parameterized). \n",
+    "To build new models, you'll need to inherit from `gpflow.models.Model`. Parameters are instantiated with `gpflow.Param`. You might also be interested in `gpflow.params.Parameterized`, which acts as a 'container' for `Param`s (for example, kernels are parameterised). \n",
     "\n",
-    "In this very simple demo, we'll implement linear multiclass classification. There will be two parameters: a weight matrix and a 'bias' (offset). The key thing to implement is the private `_build_likelihood` method, which should return a tensorflow scalar representing the (log) likelihood. By decorating the function with `@gpflow.params_as_tensors`, Param objects can be used inside `_build_likelihood`: they will appear as appropriate (constrained) tensors."
+    "In this very simple demo, we'll implement linear multiclass classification. There are two parameters: a weight matrix and a 'bias' (offset). The key thing to implement is the private `_build_likelihood` method, which returns a TensorFlow scalar that represents the (log) likelihood. If you decorate the function with `@gpflow.params_as_tensors`, you can use `Param` objects inside `_build_likelihood`; they appear as appropriate (constrained) tensors."
```

Comment:
I think the last sentence can be given a bit more meaning - verbose explanation: you can always use `Param` objects anywhere you want, but normally you would have to type, e.g., `self.kernel.variance.constrained_tensor` to get at the tf tensor to use when building a tf Graph - when a method is decorated by `@gpflow.params_as_tensors`, you only need to type `self.kernel.variance` instead, which is now the tf tensor object (and you can't access any of the other features of a Param object) - this is particularly useful when writing for example the model likelihood as there'll be lots of references to constrained_tensor attributes of various params.